tests/console/python3_new_version_check.pm

```diff
@@ -22,22 +22,20 @@ use registration "add_suseconnect_product";
 
 sub run {
     select_serial_terminal;
-    if (is_sle('>=15-SP4')) {
-        my $python_version = script_output("rpm -q python3 | awk -F \'-\' \'{print \$2}\'");
-        if ((package_version_cmp($python_version, "3.6") < 0) ||
-            (package_version_cmp($python_version, "3.7") >= 0)) {
-            # Factory default Python3 version for SLE15-SP4+ should be 3.6
-            die("Python default version differs from 3.6");
-        }
-        add_suseconnect_product('sle-module-python3');
```

Comment:
I would suggest keeping this logic for the time being for SLE15SP4+

Reply:
Because I didn't see any documents say that the default python3 version is upgraded to 3.7+